scenario-definitions.yaml

```diff
@@ -39,6 +39,7 @@ job_templates:
     settings:
       OPENQA_WEB_PROXY: 'nginx'
   openqa_install+publish:
+    <<: *common
```

Comment:
How did you notice this was broken? https://openqa.opensuse.org/tests/3570245#next_previous looks all green to me

Reply:
Ah, I should have added this link: https://openqa.opensuse.org/tests/3570330#next_previous
This only shows one test.

Reply:
Do you think we can make the error more prominent in openQA? Overall the test was working quite nicely but the error in openQA is rather subtle, right?

Reply:
I would first like to find out why the SQL query didn't work with machine set to NULL, because when I tried it manually in psql, the query returned the expected rows.